include/ensmallen_bits/sa/sa_impl.hpp

```diff
@@ -76,8 +76,9 @@ typename MatType::elem_type SA<CoolingScheduleType>::Optimize(
   size_t idx = 0;
   size_t sweepCounter = 0;
 
-  BaseMatType accept(rows, cols, arma::fill::zeros);
-  BaseMatType moveSize(rows, cols, arma::fill::none);
+  BaseMatType accept(rows, cols);
+  accept.zeros();
+  BaseMatType moveSize(rows, cols);
```

Comment:
```suggestion
  BaseMatType moveSize(rows, cols, GetFillType<BaseMatType>::none);
```
I think this is correct?